src/model/RestAction.ts

```diff
@@ -111,12 +116,14 @@ class RestAction implements Action {
                 :
                 null;
             let requestForm = this.form ? injectEvalAndVarsToMap(this.form, scenario.cache, ctx) : null;
+            let binaryData = this.dataBinary ? FS.createReadStream(this.dataBinary) : null
 
             request({
                 method:         this.method,
                 url:            injectEvalAndVarsToString(this.url, scenario.cache, ctx),
                 headers:        requestHeaders,
-                body:           requestBody,
+                body:           requestBody || binaryData,
+                encoding:       binaryData ? null : undefined,
```

Comment:
What is the purpose of setting this?

Reply:
binary data can works only if encoding = null. Otherwise 'requests' sets 'utf8' by default and the request fails